packages/tools/fluid-runner/README.md

```diff
@@ -1,10 +1,16 @@
 # @fluidframework/fluid-runner
 
-This package contains utility for running various functionality inside a Fluid Framework environment.
+This package contains utility for running various functionality inside an offline Fluid Framework environment.
+Currently, the only function of the fluid-runner tool is `exportFile`. The tool may be expanded for futher functionality.
```

Comment:
"may be expanded for further functionality" -- does this mean we will be adding to it?  Or that we've provided extensibility points for a user to expand on it?  Prob clarify with either:

```suggestion
Currently, the only function of the fluid-runner tool is `exportFile`. We'll be adding more functionality in the future.
```

OR

```suggestion
Currently, the only function of the fluid-runner tool is `exportFile`. However, you can plug in your own functionality by following these steps ________...
```